.changeset/breezy-tigers-attend.md

```diff
@@ -0,0 +1,24 @@
+---
+'@openfn/language-dhis2': major
+---
+
+Migrates the adaptor to the new Tracker API (v36+) for `trackedEntities`,
+`enrollments`, `events` and `relationships`.
+
+This release is designed for compatibility with DHIS2 v42, which drops support
+for a number of endpoints.
+
+The `create`, `update`, `upsert` and `destroy` functions will automatically map
```

Comment:
This is out of date after your changes @mtuchi - can you please review and update these change notes?

Reply:
Updated changeset